taxi/urls.py

```diff
@@ -42,7 +47,17 @@
     path("cars/create/", CarCreateView.as_view(), name="car-create"),
     path("cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
     path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
+    path("cars/<int:pk>/driver_update/", car_driver_add, name="driver-update"),
+    path("cars/<int:pk>/driver_delete/", car_driver_delete,
+         name="driver_delete"),
```

Comment:
Consider renaming the `driver-update` path to something more descriptive, such as `car-driver-add`, to match the function name `car_driver_add`. Similarly, ensure that the `driver_delete` name matches the function name `car_driver_delete` for consistency.